.github/DOCUMENTATION_TEMPLATE.md

```diff
@@ -0,0 +1,66 @@
+# How to contribute to MFTF docs
+
+MFTF documentation is kept within the /docs/ folder in this repository.
+We welcome contributions to the documentation.
+This page describes the process for submitting documentation and serves as a template for a properly written content.
+
+The contribution workflow for the Magento functional testing framework (MFTF) documentation is the same as submitting code.
+
+1. Create a branch from the `develop` branch in the [MFTF repository][].
+1. Make edits/additions/deletions as needed.
+1. Submit your PR to the `develop` branch.
+
+Once submitted, a member of the documentation team will review and merge it.
+We will inform you if it needs any additional processing.
+
+The documentation in this repository is used as the source for the [MFTF documentation][].
+Any changes to the Table of Contents will need to be made via a separate PR in the regular [Magento Developer documentation repository][].
```

Comment:
1. Replace _via_ with other word.

   See DevDocs style guide:
   > Non-English words and weak phrases
   > Some non-English words and abbreviations are difficult to translate, and some users might be unfamiliar with them.

   > Avoid the following common words and phrases

   > Info
   > Etc. or etcetera
   > e.g.
   > Misc. or miscellaneous
   > Via
   > Make sure
2. It is not clear why _Table of Contents_ is capitalized.
3. Replace _PR_ with _pull request_. Avoid abbreviations. GitHub interface doesn't use this abbreviation, so it can be not clear (GitHub article [About pull requests](https://help.github.com/en/articles/about-pull-requests))

   See DevDocs style guide:

   > Acronyms
   > Use only the most obvious acronyms. For instance, use PHP, HTML, XML for the names of those computer languages, not an unknown acronym. Try to avoid acronyms in your text when possible.

Reply:
Fixed.